src/router.js

```diff
@@ -15,7 +15,9 @@ const router = (request, response)=> {
             handlers.publicHandler(request,response)
     }
     //-----ADD POST Endpoint-----------------------------------------------------
-    // to do get cities and set data 
+    else if (request.url === '/addpost'){
+            handlers.addPostHandler(request,response)
```

Comment:
It's better if you check for the request method as well such as `if (request.url === '/addpost' && request.method === 'POST'`.